contractcourt/channel_arbitrator.go

```diff
@@ -600,8 +626,9 @@ func (t transitionTrigger) String() string {
 // the appropriate state transition if necessary. The next state we transition
 // to is returned, Additionally, if the next transition results in a commitment
 // broadcast, the commitment transaction itself is returned.
-func (c *ChannelArbitrator) stateStep(triggerHeight uint32,
-	trigger transitionTrigger) (ArbitratorState, *wire.MsgTx, error) {
+func (c *ChannelArbitrator) stateStep(
+	triggerHeight uint32, trigger transitionTrigger,
+	confCommitSet *CommitSet) (ArbitratorState, *wire.MsgTx, error) {
```

Comment:
`confCommitSet` unused (in this commit).

Reply:
Adding it in this commit made it easier to construct the commits that came after it. I don't think there's much to be gained by moving around this small portion of the diff.